### PR TITLE
Revised deployment topics for .NET Core 2.1

### LINKED ISSUE #5398: Update .NET Core deployment topic
# Update .NET Core deployment topic

Update to [.NET Core application deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/)

The topic requires some revision for both .NET Core 2.0 and .NET Core 2.1 changes. At a minimum, these include:

- For .NET Core 2.0: Starting with Visual Studio 15.7, RIDs for self-contained deployments do not have to be set in the project file; they can be selected from the publish settings page, or they can be added to pubxml.
- For .NET Core 2.0: Visual Basic is now supported.
- For .NET Core 2.0: Globalization invariant mode can reduce the size of a self-contained deployment.
- For .NET Core 2.1: Self-contained application servicing

### PR DESCRIPTION
**Pull request by rpetrusha**
## Revised deployment topics for .NET Core 2.1

Fixes #5398 

Related to dotnet/samples#284

### CODE REVIEW

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

File: docs/core/deploying/deploy-with-cli.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: .NET Core app deployment with CLI tools
 description: Learn .NET Core app deployment with command-line interface (CLI) tools
 author: rpetrusha
 ms.author: ronpet
-ms.date: 04/18/2017
+ms.date: 00/03/2018
```
Comment (mairaw) [inline docs/core/deploying/deploy-with-cli.md]: wrong date here

**Review by BillWagner — APPROVED** [review]
This is a great update @rpetrusha 

Very clear explanations all the way through. I did notice one discrepency for unix based systems After clarifying that, it's ready to merge.

**Pull request merged**